src/config/entities/configuration.ts

```diff
@@ -41,6 +43,27 @@ export default () => ({
           8453: { chainName: 'base' },
         },
       },
+      zerion: {
+        baseUri: process.env.ZERION_BASE_URI || 'https://api.zerion.io',
+        apiKey: process.env.ZERION_API_KEY,
+        chains: {
```

Comment:
Are these names standardised? We should consider if the Safe Config Service should play a role with storing these names.

Reply:
They are not. These are the names the providers use on their APIs, they don't map 1:1 to the Safe Config Service ones, or ERC-3770. 

As a follow-up, we could consider if it makes sense to push this configuration to the Config Service 👍🏻

Reply:
Reading that there are not standardised, nor match ERC-3770 short names, ignore suggestion to [rename them](https://github.com/safe-global/safe-client-gateway/pull/1096#discussion_r1475934308).

Reply:
> As a follow-up, we could consider if it makes sense to push this configuration to the Config Service 👍🏻

I'd say that your solution is totally fine. We can re-evaluate adding this to the Safe Config Service but given that they are not standardised, this also seems to be a good place to have the configuration.